plugins/org.jkiss.dbeaver.data.office/OSGI-INF/l10n/bundle_ja.properties

```diff
@@ -17,6 +17,8 @@ dataTransfer.processor.xlsx.property.splitSqlText.description=CR\u306B\u3088\u30
 dataTransfer.processor.xlsx.property.headerFont.name=\u30D8\u30C3\u30C0\u30FC\u884C\u30D5\u30A9\u30F3\u30C8
 dataTransfer.processor.xlsx.property.headerFont.description=\u6700\u521D\u306E\u884C\u306E\u30D5\u30A9\u30F3\u30C8\u30D7\u30ED\u30D1\u30C6\u30A3
 dataTransfer.processor.xlsx.property.border.name=\u30DC\u30FC\u30C0\u30FC\u30B9\u30BF\u30A4\u30EB
+dataTransfer.processor.xlsx.property.headerFontName.name=\u000d\u000a\u30d8\u30c3\u30c0\u30fc\u30d5\u30a9\u30f3\u30c8\u540d
+dataTransfer.processor.xlsx.property.headerFontName.description=\u6700\u521d\u306e\u884c\u306e\u30d5\u30a9\u30f3\u30c8\u540d
```

Comment:
Until you're sure that you know that language, such translations are useless and tend to contain grammatical errors.